appveyor.yml

```diff
@@ -0,0 +1,66 @@
+
+platform: x64
+
+environment:
+ matrix:
+  - DC: dmd
+    DVersion: 2.071.0
+    arch: x64
+
+skip_tags: true
+
+branches:
+  only:
+    - windows
+
+install:
+  - ps: function SetUpDCompiler
+        {
+          if($env:arch -eq "x86"){
+            $env:DConf = "m32";
+          }
+          elseif($env:arch -eq "x64"){
+            $env:DConf = "m64";
+          }
+          echo "downloading ...";
+          $env:toolchain = "msvc";
+          $version = $env:DVersion;
+          Invoke-WebRequest "http://downloads.dlang.org/releases/2.x/$($version)/dmd.$($version).windows.7z" -OutFile "c:\dmd.7z";
+          echo "finished.";
+          pushd c:\\;
+          7z x dmd.7z > $null;
+          popd;
+        }
+  - ps: SetUpDCompiler
+  - powershell -Command Invoke-WebRequest https://code.dlang.org/files/dub-1.0.0-beta.1-windows-x86.zip -OutFile dub.zip
+  - 7z x dub.zip -odub > nul
+  - set PATH=%CD%\%binpath%;%CD%\dub;%PATH%
+  - dub --version
+
+before_build:
+  - ps: if($env:arch -eq "x86"){
+            $env:compilersetupargs = "x86";
+            $env:Darch = "x86";
+          }
+        elseif($env:arch -eq "x64"){
+            $env:compilersetupargs = "amd64";
+            $env:Darch = "x86_64";
+        }
+  - ps : if($env:DC -eq "dmd"){
+           $env:PATH += ";C:\dmd2\windows\bin;";
+         }
+         elseif($env:DC -eq "ldc"){
+           $version = $env:DVersion;
+           $env:PATH += ";C:\ldc2-$($version)-win64-msvc\bin";
+           $env:DC = "ldc2";
+         }
+  - ps: $env:compilersetup = "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall";
+  - '"%compilersetup%" %compilersetupargs%'
```

Comment:
It seems like it is common practice to pack all that scripts inside `appveyor.yml`. I took that script from [https://wiki.dlang.org/Continuous_Integration](https://wiki.dlang.org/Continuous_Integration). Anyway, I distilled the scripts to bare minimum. Is it good enough or should I move the rest to the separate file.

The problem with separate files is the scripts sets and references environmental variables heavily, and I'm not sure how it would work on the subscript boundary.